riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/RiptideSettings.java

```diff
@@ -23,6 +23,11 @@
     private Defaults defaults = new Defaults();
     private GlobalOAuth oauth = new GlobalOAuth();
     private Map<String, Client> clients = new LinkedHashMap<>();
+    private Boolean mocked = false;
```

Comment:
Why not `boolean`?

Reply:
Putting this here technically allows users to do this:

```yaml
riptide:
  mocked: true
```

Reply:
We can unbox - no particular reason to use Boolean